docs/roman/outlier_detection/outlier_detection_step.rst

```diff
@@ -0,0 +1,21 @@
+.. _outlier_design:
+
+OutlierDetectionStep
+--------------------
+
+This module provides the sole interface to all methods of performing outlier detection
+on Roman observations. The outlier detection algorithm used for WFI data is implemented
+in :py:class:`~romancal.outlier_detection.outlier_detection.OutlierDetection`
+and described in :ref:`outlier-detection-imaging`.
+
+.. note::
+    Whether the data are being provided in an ASN file or as a list of ASDF filenames,
```

Comment:
I'd not use the ASN acronym here and would spell it out as association file and link to the asn_from_list docs. 
https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/asn_from_list.html